plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -367,6 +367,50 @@ func limitedByDefault(usage api.ResourceList, limitedResources []resourcequotaap
 	return result
 }
 
+func matchScope(priorityClass string, limitedResources []resourcequotaapi.LimitedResource) bool {
+	match := false
+	for _, limitedResource := range limitedResources {
```

Comment:
Isn't this check only make sense for `resource: pods`?

Reply:
there are other scopes besides these that we would need to handle.

is there a reason we cant reuse the pod scopes func so it handles BestEffort, NotBestEffort, Terminating, NotTerminating matches as well?  the admin could configure this to use any valid scope.